### ISSUE TITLE
Not working with Eclipse Theia

### ISSUE THREAD

**Issue by chroberino** [tjx666/open-in-external-app#63]
### Description

I don't get any context menu items when using this extension with Eclipse Theia. I have tried with both, the latest ready-made beta version of "Theia IDE" (Windows) as well as a self-built Electron app based on Theia 1.45.0 (Windows), but it is the same behaviour in both: after installing the "Open in External App" extension from the Open VSX Registry I don't get the option to open files in the default external app.

### Extension Version

0.10.0

### VS Code Version

1. Eclipse Theia IDE 1.47.100
2. Eclipse Theia 1.45.0 sources 

### Extension Settings

No config, just want to use the default application as configured in the OS.

### Logs, Notifications, Screenshots, Screen Recording etc

There is no "Open in External App" channel in the Output panel.

**Comment by tjx666** [comment]
Other vscode extensions can be used in eclipse theia?

Did you set theia correctlly as described in https://theia-ide.org/docs/authoring_vscode_extensions#prerequisites-for-running-vs-code-extensions-in-theia?

**Comment by chroberino** [comment]
> Other vscode extensions can be used in eclipse theia?
> 
> Did you set theia correctlly as described in https://theia-ide.org/docs/authoring_vscode_extensions#prerequisites-for-running-vs-code-extensions-in-theia?

Yes, I can confirm that other extensions (both pre-bundled with my self-built Theia as well as those installed via Open VSX Registry) are working. Actually I can even see the settings of the "Open in External App" extension within Theia. And also the command palette reveals the extension is installed. Please see screenshots.
However, the conext-menu item when right-clicking on a file in the Explorer is missing.

![grafik](https://github.com/tjx666/open-in-external-app/assets/140696778/0c2d751b-ae7d-48b4-98f3-e519b31e5d4b)
![grafik](https://github.com/tjx666/open-in-external-app/assets/140696778/d5344be2-5d88-4d37-aa57-4df1bf089e9a)
![grafik](https://github.com/tjx666/open-in-external-app/assets/140696778/7e4b103b-e9af-4140-b389-59a0a87b04dd)

**Comment by chroberino** [comment]
Just FYI: https://github.com/eclipse-theia/theia/issues/13535
The `vscode.env.openExternal` call in Theia is also not working as expected.
